src/bundle/Resources/views/content/tabs/details.html.twig

```diff
@@ -0,0 +1,89 @@
+<div class="ez-list-toolbar">
+    <h2 class="ez-list-toolbar-label">{{ 'locationview.details.title'|trans({}, 'locationview') }}</h2>
+</div>
+
+<table class="ez-table-data">
+    <thead>
+        <tr>
+            <th>{{ 'locationview.details.creator'|trans({}, 'locationview') }}</th>
+            <th>{{ 'locationview.details.lastcontributor'|trans({}, 'locationview') }}</th>
+            <th>{{ 'locationview.details.versions'|trans({}, 'locationview') }}</th>
+            <th>{{ 'locationview.details.translations'|trans({}, 'locationview') }}</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>{{ ez_render_field(creator, 'first_name') }} {{ ez_render_field(creator, 'last_name') }} ({{ contentInfo.publishedDate|localizeddate('medium', 'short', app.request.locale) }})</td>
+            <td>{{ ez_render_field(lastContributor, 'first_name') }} {{ ez_render_field(lastContributor, 'last_name') }} ({{ versionInfo.modificationDate|localizeddate('medium', 'short', app.request.locale) }})</td>
+            <td>{{ versionInfo.versionNo }}</td>
+            <td>{{ translations|length }}
+                {% for translation in translations %}
+                    {{ translation.name }}<br>
+                {% endfor %}
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+<div class="ez-list-toolbar">
+    <h2 class="ez-list-toolbar-label">{{ 'locationview.details.technicaldetails'|trans({}, 'locationview') }}</h2>
+</div>
+
+<table class="ez-table-data">
```

Comment:
@bdunogier - Label change - 'Id' should be replaced with 'ID' for all labels in Technical and Section details table